web/app/scripts/src/services/api.es.js

```diff
@@ -33,6 +33,12 @@ module.service(moduleProperties.componentName, [
             getForm: function (url) {
                 return $http.get(url)
                     .then((response) => response.data);
+            },
+            post: function (baseUrl, data) {
+                const endpoint = 'respond';
```

Comment:
I don't think it's a better solution than this, it's still out of sync with backend. A better solution would be to pass in the endpoints from the backend somehow.